impeller/entity/contents/runtime_effect_contents.cc

```diff
@@ -26,7 +26,7 @@ namespace impeller {
 
 // static
 BufferView RuntimeEffectContents::EmplaceVulkanUniform(
-    std::vector<uint8_t>& input_data,
+    const std::shared_ptr<std::vector<uint8_t>>& input_data,
```

Comment:
This is still passing in a non-const vector =)  You should be able to freely cast it to a const vector here.  I was thinking `const std::vector<uint8_t>*` would be easiest, doing the shared_ptr ref is fine too.

Reply:
now its really const